plume-models/src/users.rs

```diff
@@ -202,30 +204,6 @@ impl User {
             .map_err(Error::from)
     }
 
-    pub fn update(
```

Comment:
so this whole function was useless? Some other must be too then

Reply:
Yes, I would like to do a big code cleanup one day, removing all `pub`, progressively reintroducing them until it compiles, and remove everything that Rust says is unused.